# The RtE Id defined in the emf model
string id
string name
string type
RtEValue[] values